.github/workflows/pull.yml

```diff
@@ -131,6 +131,31 @@ jobs:
         PYTHON_EXECUTABLE=python bash .ci/scripts/setup-linux.sh "${BUILD_TOOL}"
         PYTHON_EXECUTABLE=python bash .ci/scripts/test_quantized_aot_lib.sh
 
+  test-pybind-build:
+    name: test-pybind-build
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    strategy:
+      matrix:
+        include:
+          - build-tool: cmake
+      fail-fast: false
+    with:
+      runner: linux.2xlarge
+      docker-image: executorch-ubuntu-22.04-clang12
+      submodules: 'true'
+      ref: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.sha || github.sha }}
+      timeout: 90
+      script: |
+        # The generic Linux job chooses to use base env, not the one setup by the image
+        CONDA_ENV=$(conda env list --json | jq -r ".envs | .[-1]")
+        conda activate "${CONDA_ENV}"
+
+        # build module for executorch.extension.pybindings.portable_lib
+        PYTHON_EXECUTABLE=python EXECUTORCH_BUILD_PYBIND=ON bash ./install_requirements.sh
```

Comment:
I think you need to use `setup-linux` here instead https://github.com/pytorch/executorch/blob/main/.github/workflows/_unittest.yml#L37.  The change in `install_requirements.sh` looks out of place and probably not needed.  `setup-linux` targets CI job exclusively.